Zig example broken

https://github.com/im-tomu/fomu-toolchain/runs/3501436881?check_suite_focus=true
```
RISC-V Zig Example
  + cd riscv-zig-blink
  + zig build
  .\src\fomu.zig:57:34: error: container 'std.io' has no member called 'Writer'
  pub const messibleWriter = std.io.Writer(void, WriteError, messibleWrite){ .context = {} };
                                   ^
  .\src\fomu.zig:43:5: note: referenced here
      messibleWriter.print("PANIC: {s}\r\n", .{message}) catch void;
      ^
  riscv-zig-blink...The following command exited with error code 1:
  C:\ProgramData\Chocolatey\lib\zig\tools\zig-windows-x86_64-0.6.0\zig.exe build-exe D:\a\fomu-toolchain\fomu-toolchain\riscv-zig-blink\src\main.zig --cache-dir D:\a\fomu-toolchain\fomu-toolchain\riscv-zig-blink\zig-cache --name riscv-zig-blink -target riscv32-freestanding-none -mcpu generic_rv32 --linker-script D:\a\fomu-toolchain\fomu-toolchain\riscv-zig-blink\ld\linker.ld --cache on 
  
  Build failed. The following command failed:
  D:\a\fomu-toolchain\fomu-toolchain\riscv-zig-blink\zig-cache\o\iitzuNv-_qFTArx5yWq3vlJWPHN3M0w1Kwt8wi5aRZ1UXEigtZDnYiSfBEFKUyIg\build.exe C:\ProgramData\Chocolatey\lib\zig\tools\zig-windows-x86_64-0.6.0\zig.exe D:\a\fomu-toolchain\fomu-toolchain\riscv-zig-blink D:\a\fomu-toolchain\fomu-toolchain\riscv-zig-blink\zig-cache
```

--- a/.github/workflows/update.yml
+++ b/.github/workflows/update.yml
@@ -82,14 +82,14 @@ jobs:
       if: matrix.os == 'ubuntu'
       run: |
         tar xzf ${{ env.FOMU_TOOLCHAIN }}.tar.gz
-        curl -L https://ziglang.org/download/0.6.0/zig-linux-x86_64-0.6.0.tar.xz | tar -xJf -
-        echo "$(pwd)/zig-linux-x86_64-0.6.0" >> $GITHUB_PATH
+        curl -L https://ziglang.org/download/0.8.0/zig-linux-x86_64-0.8.0.tar.xz | tar -xJf -
+        echo "$(pwd)/zig-linux-x86_64-0.8.0" >> $GITHUB_PATH
 
     - name: Install (Windows)
       if: matrix.os == 'windows'
       run: |
         unzip ${{ env.FOMU_TOOLCHAIN }}.zip
-        choco install zig --version 0.6.0
+        choco install zig --version 0.8.0
         ln -s $(which python) /usr/bin/python3
 
     - name: Install (Mac OS)

